avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/serialization/AVRO_1_8/StringableRecord_SpecificSerializer_6174384286732341990.java

```diff
@@ -62,6 +62,16 @@ public void serializeStringableRecord0(StringableRecord data, Encoder encoder)
         serializeStringableSubRecord0(subRecord0, (encoder));
         AnotherSubRecord subRecordWithSubRecord0 = ((AnotherSubRecord) data.get(8));
         serializeAnotherSubRecord0(subRecordWithSubRecord0, (encoder));
+        String stringUnion0 = ((String) data.get(9));
+        if (stringUnion0 == null) {
+            (encoder).writeIndex(0);
+            (encoder).writeNull();
+        } else {
+            if (stringUnion0 instanceof String) {
```

Comment:
Unnecessary check?

Reply:
It  is not necessary..
I noticed this inefficiency, but I didn't fix it because of laziness..
Thanks for pushing this optimization.